studio/components/grid/types/grid.ts

```diff
@@ -12,63 +12,34 @@ export interface GridProps {
 }
 
 export interface SupabaseGridProps {
-  /**
-   * database table swagger or table name
-   */
+  // database table swagger or table name
```

Comment:
Any reason for changing these comment types? I think the editor can read `/**` comments and show tooltips when we're developing

Reply:
ahhh i see - no real reason tbh, i'll revert it